tests/lab_test.py

```diff
@@ -107,8 +107,8 @@ def test_make_status(self):
         self.assertEqual(open[keys[0]], 'open')
 
         for key in keys[1:]:
-            self.assertEqual(closed[key], 0)
-            self.assertEqual(open[key], 1)
+            self.assertEqual(closed[key], '0')
+            self.assertEqual(open[key], '1')
```

Comment:
These are all cast to strings here: https://github.com/Pitt-CSC/PittAPI/pull/82/files#diff-1c156c9a7cf25c89ef4fd8323d5c4d35L66